ex-submodules/rapidsms-logistics/logistics/models.py

```diff
@@ -1482,7 +1482,7 @@ def get_product(self, product_code):
         if the product can't be found.
         """
         try:
-            return Product.objects.get(sms_code__icontains=product_code)
+            return Product.objects.get(sms_code__iexact=product_code)
```

Comment:
this seems like a good change. out of curiosity, do you remember which product / scenario specifically tripped the error that this fixed?

Reply:
The scenario that produced this issue is one where we were reporting stock on hand for the products below. The use of "contains" using either product code "cm" or "cf" would result in more than one record being returned which would then generate an error identifying one of the product codes are not existing

- Condom-Male with product code **cm**
- Condom-Female with product code **cf**
- Socially Marketed Condom-Male with product code **mcm**
- Socially Marked Condom-Female with product code **mcf**